recipes/rnabob/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+make clean
+make
+make install HOME=/usr/local
```

Comment:
The ```make install``` tries to use $HOME to copy the executable.

```
BINDIR 		= $(HOME)/bin
MANDIR          = $(HOME)/man
```

```
PROGS=     rnabob
```

```
install: $(PROGS)
	cp $(PROGS)   $(BINDIR)/
	cp rnabob.man $(MANDIR)/man1/rnabob.1
```

So, I'm giving the value it should have for home. And in the patch I removed the second line.